conf/jetty/jetty.xml.production

```diff
@@ -614,7 +614,73 @@
 		    <Set name="replacement">%%zimbraMailURL%%</Set>
 		</New>
 	    </Arg>
-        </Call>       
+        </Call>      
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">/bundle.*</Set>
+              <Set name="name">Cache-Control</Set>
+              <Set name="value">Max-Age=86400,public</Set>
+            </New>
+          </Arg>
+        </Call>
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">.*(.chunk.*)</Set>
+              <Set name="name">Cache-Control</Set>
+              <Set name="value">Max-Age=31536000,public</Set>
+              <Set name="terminating">true</Set>
```

Comment:
check if terminating is required or not
check for all rewrites

Reply:
done